platform/commonUI/edit/res/templates/edit-action-buttons.html

```diff
@@ -42,11 +42,12 @@
     </span>
 
     <span ng-repeat="currentAction in otherEditActions">
-        <a class='s-button {{currentAction.getMetadata().cssclass}}'
-           title='{{currentAction.getMetadata().name}}'
-           ng-click="currentAction.perform()"
-           ng-class="{ major: $index === 0 && saveActions.length === 0 }">
-            <span class="title-label">{{currentAction.getMetadata().name}}</span>
-        </a>
+        <mct-control key="'button'"
+                     structure="{
+                        text: currentAction.getMetadata().name,
+                        click: currentAction.perform,
```

Comment:
I'd do this as `currentAction.perform.bind(currentAction)` so that the action being invoked has an appropriate `this`. I would expect this to work currently because LoggingActionDecorator reassigns `perform` on actions at the instance level, but if that is ever refactored/removed it could cause problems here.